plugins/tricatworld/build.gradle

```diff
@@ -0,0 +1,21 @@
+plugins {
+    id 'java'
+}
+
+group 'de.dfki'
+version '4.0.1'
+
+sourceCompatibility = 1.8
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    testCompile group: 'junit', name: 'junit', version: '4.12'
+    compile project(":core")
+    compile project(":plugins:extensionAPI")
+    compile project(":plugins:ssi")
+    implementation fileTree('lib')  // Needed to use Charamelapi-1.0.jar
```

Comment:
Can you import as single flie?

Reply:
Sure. Changes for `reeti`, `caramel` and `tricatworld` plugins.